spec/models/equipment_model_spec.rb

```diff
@@ -151,6 +151,51 @@
     end
   end
 
+  describe 'counter cache' do
+    before(:each) do
+      @model = FactoryGirl.create(:equipment_model)
+      @res = FactoryGirl.build(:checked_out_reservation,
+                               equipment_model: @model, 
+                               start_date: Time.zone.today - 2.days,
+                               due_date: Time.zone.today - 1.day)
+      @res.save(validate: false)
+    end
+    it 'increments when a reservation is marked as overdue' do
+      @res.update_attribute(:overdue, true)
+      expect(@model.overdue_count).to eq 1
+    end
+    it 'only increments once per reservation' do
+      @res.update_attribute(:overdue, true)
+      expect(@model.overdue_count).to eq 1
+      @res.update_attribute(:notes, 'test')
+      expect(@model.overdue_count).to eq 1
+    end
+    it 'decrements when an overdue reservation is checked in' do
+      @res.update_attribute(:overdue, true)
+      expect(@model.overdue_count).to eq 1
+      @res.update_attributes(FactoryGirl.attributes_for(
+        :overdue_returned_reservation, equipment_model: @model))
+      expect(@model.overdue_count).to eq 0
+    end
+    it 'only decrements once per reservation' do
+      @res.update_attribute(:overdue, true)
+      expect(@model.overdue_count).to eq 1
+      @res.update_attributes(FactoryGirl.attributes_for(
+        :overdue_returned_reservation, equipment_model: @model))
+      expect(@model.overdue_count).to eq 0
+      @res.update_attribute(:notes, 'test')
+      expect(@model.overdue_count).to eq 0
```

Comment:
Same here, we can check `not_to change` instead (which reads better to me). This applies to the test on line 167 as well.